tests/build_rpms.sh

```diff
@@ -61,3 +64,12 @@ mkdir -p htcondor-ce/travis_deploy
 cp -f /tmp/rpmbuild/RPMS/*/*.rpm htcondor-ce/travis_deploy/
 cp -f /tmp/rpmbuild/SRPMS/*.rpm htcondor-ce/travis_deploy/
 
+# Install the python3 rrdtool manually, must be compiled from source to import correctly
+git clone -q https://github.com/commx/python-rrdtool
+pushd python-rrdtool
+python3 setup.py install
+popd
+
+# Manually install the required python3 packages
+# TODO: These should be getting installed from the requirements.txt file.
+pip3 install flask genshi CherryPy==3.*
```

Comment:
It's also a little concerning that this is required, as all of this should be brought in by the RPM packaging if we get the dependencies right.